pkg/util/log/clog.go

```diff
@@ -775,6 +783,14 @@ func (l *loggingT) outputLogEntry(s Severity, file string, line int, msg string)
 	var stacks []byte
 	var fatalTrigger chan struct{}
 	if s == Severity_FATAL {
+		fatalChanMu.Lock()
```

Comment:
The logger is already locked above (l.mu.lock) so I think this lock is unneeded.